### DISCUSSION TITLE
Mainsail / Klipper layer status 0 of 0

### DISCUSSION THREAD

**Discussion by Glenn1963** [OrcaSlicer/OrcaSlicer#1522] [General]
![image](https://github.com/SoftFever/OrcaSlicer/assets/17497231/99ca9d92-89fe-4a11-bc79-d605da80c820)

This is really minor, but I can't figure out why in OrcaSlicer the layer status doesn't work, but it does in SuperSlicer. I have looked at the SS g-code but cannot see anything that would cause that to be updated. 

I have added the M117 command to display the values, so it's no big deal, but I am curious.

Thanks

**Comment by Tiwatz** [comment]
Did you find a solution to this? I just recently got the same issue.

**Reply by Glenn1963** [reply]
I haven't, but I haven't tried the 2.0 version either.

**Comment by 5aaee9** [comment]
Add the gcode from https://docs.mainsail.xyz/overview/slicer/superslicer#best-practise-g-codes resolve this problem
